src/audio/kpb.c

```diff
@@ -51,10 +51,15 @@
 static const struct comp_driver comp_kpb;
 
 LOG_MODULE_REGISTER(kpb, CONFIG_SOF_LOG_LEVEL);
-
+#if CONFIG_IPC_MAJOR_4
+/* A8A0CB32-4A77-4DB1-85C7-53D7EE07BCE6 */
+DECLARE_SOF_RT_UUID("kpb", kpb_uuid, 0xA8A0CB32, 0x4A77, 0x4DB1,
+		    0x85, 0xC7, 0x53, 0xD7, 0xEE, 0x07, 0xBC, 0xE6);
+#else
 /* d8218443-5ff3-4a4c-b388-6cfe07b9562e */
 DECLARE_SOF_RT_UUID("kpb", kpb_uuid, 0xd8218443, 0x5ff3, 0x4a4c,
-		 0xb3, 0x88, 0x6c, 0xfe, 0x07, 0xb9, 0x56, 0x2e);
+		    0xb3, 0x88, 0x6c, 0xfe, 0x07, 0xb9, 0x56, 0x2e);
+#endif
```

Comment:
Hmm, it seems the none of IPC4 topologies were using this yet, so no topology update is needed to change (for IPC4).